### PR TITLE
Add the description of the model's attribute.

### PR DESCRIPTION
**Pull request by wisererik**

### CODE REVIEW

File: openapi-spec/swagger.yaml [hunk 1]
```diff
@@ -732,9 +732,9 @@ definitions:
             type: string
   Pool:
     description: >-
-      A backend is initialized by specific driver configuration. Each backend
-      can be regarded as a docking service between SDS controller and storage
-      service.
+      A pool is discoveried and updated by a dock servie. Each pool can be regarded
```
Comment (leonwanghui) [inline openapi-spec/swagger.yaml]: it should be changed to ```service```

File: pkg/model/dock.go [hunk 2]
```diff
@@ -24,11 +24,25 @@ package model
...
+	// The storage type of the dock.
+	// One of: "block", "file" or "object".
 	StorageType string `json:"storageType,omitempty"`
-	Endpoint    string `json:"endpoint,omitempty"`
-	DriverName  string `json:"driverName,omitempty"`
+
+	// Endpoint represents the dock server's access address.
+	Endpoint string `json:"endpoint,omitempty"`
+
+	// DriverName represents the dock provider, such as Ceph, LVM and etc.
```
Comment (leonwanghui) [inline pkg/model/dock.go]: Currently One of: "cinder", "ceph", "lvm" and "default".

File: pkg/model/pool.go [hunk 3]
```diff
@@ -19,22 +19,42 @@ This module implements the common data structure.
 
 package model
 
-// A backend is initialized by specific driver configuration. Each backend
-// can be regarded as a docking service between SDS controller and storage
-// service.
+// A pool is discoveried and updated by a dock servie. Each pool can be regarded
```
Comment (leonwanghui) [inline pkg/model/pool.go]: ```service``` here.

File: openapi-spec/swagger.yaml (hunk 1 above)
Comment (wisererik) [reply openapi-spec/swagger.yaml]: @leonwanghui sorry, what does the service refer to here?

Comment (leonwanghui) [reply openapi-spec/swagger.yaml]: I mean there is a spelling error in your patch, just change ```servie``` to ```service```.

**Comment by wisererik** [comment]
@leonwanghui tks, I have resolved it.

**Comment by leonwanghui** [comment]
Good job!

**Pull request merged**